language/move-lang/src/test_utils/mod.rs

```diff
@@ -26,20 +26,16 @@ impl std::error::Error for StringError {
     }
 }
 
-pub fn stdlib_files() -> Vec<&'static str> {
-    let mut files: Vec<&'static str> = vec![];
+pub fn stdlib_files() -> Vec<String> {
     let dirfiles = datatest_stable::utils::iterate_directory(Path::new(STD_LIB));
-    let filtered = dirfiles.filter(|path| {
-        path.extension()
-            .map(|ext| ext.to_str().map(|s| s == "move").unwrap_or(false))
-            .unwrap_or(false)
-    });
-    for path in filtered {
-        let s: String = path.into_os_string().into_string().unwrap();
-        let s: &'static str = Box::leak(Box::new(s));
-        files.push(s)
-    }
-    files
+    dirfiles
+        .filter(|path| {
+            path.extension()
+                .map(|ext| ext.to_str().map(|s| s == "move").unwrap_or(false))
+                .unwrap_or(false)
+        })
+        .map(|path| path.into_os_string().into_string().unwrap())
```

Comment:
Use `filter_map` instead?

Reply:
I don't think filter_map would be cleaner here, as the original check is a boolean and does not produce a `Option<Path>`

In other words, I think using filter_map would just wrap the map lambda in an if/else and not really make it more readable.

Reply:
But maybe there is some clever way to check the extension while converting it? unsure

Reply:
I have found the clever way